portfolio/src/main/webapp/aboutme.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Sydney Beasley Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyA_WLXfgBcN3n6ePBixMezUK6PQLeBLA84"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="createMap();">
+    <ul>
+      <li><a href="index.html">Home</a></li>
```

Comment:
A suggestion - it's nice to give the current navigation item an "active" visual state. An easy way to do it is not to add the link around the item for the current page. A user wouldn't need to navigate to the page they're on anyway.